resources/shuttle-aws-rds.mdx

```diff
@@ -16,4 +16,23 @@ Add `shuttle-aws-rds` to the dependencies for your service. Every engine is behi
 | MySql    | mysql        | shuttle_aws_rds::MySql    |
 | MariaDB  | mariadb      | shuttle_aws_rds::MariaDB  |
 
+## Example
+The code of the main struct would look as follows:
+```
```

Comment:
To get rust syntax highlighting and also show the filename as a tab in the snippet, you can do this (this should be done for all the resource examples):
```suggestion
```rust lib.rs
```